app/common/data/models/__init__.py

```diff
@@ -74,4 +76,49 @@ class CollectionSchema(BaseModel):
     created_by_id: Mapped[uuid.UUID] = mapped_column(ForeignKey("user.id"))
     created_by: Mapped[User] = relationship("User")
 
+    sections: Mapped[list["Section"]] = relationship(
+        "Section", lazy=True, order_by="Section.order", collection_class=ordering_list("order", count_from=1)
```

Comment:
Do we want `count_from=1`? Are we more used to 0-indexed lists? We add this and then we have a bunch of logic to do `- 1` stuff when reordering; if we remove this does it remove the need for that juggling?

Reply:
It does, but we use this to set the `order` which I'm expecting we will display to users at some point when we show a list of questions (eg. all questions) and some of the tasklists in pre-award had numbers by them... so starting with 1. seemed easier for that?

Reply:
Agreed to start at 0, still todo